docker-java-api/src/main/java/com/github/dockerjava/api/command/CreateConfigCmd.java

```diff
@@ -0,0 +1,29 @@
+package com.github.dockerjava.api.command;
+
+import com.github.dockerjava.api.exception.ConflictException;
+import com.github.dockerjava.api.model.ConfigSpec;
+
+import javax.annotation.CheckForNull;
+
+/**
+ * Command to create a new config
+ *
+ * @since {@link RemoteApiVersion#VERSION_1_30}
+ */
+public interface CreateConfigCmd extends SyncDockerCmd<CreateConfigResponse> {
+
+    @CheckForNull
+    ConfigSpec getConfigSpec();
```

Comment:
since there is no nested "ConfigSpec" object in https://docs.docker.com/engine/api/v1.40/#operation/ConfigCreate, could you please align it with the API?

Reply:
I have removed the nested "ConfigSpec". I had introduced this "ConfigSpec" to be in line with the Docker Secret implementation.